build/charts/yorkie-cluster/templates/istio/ratelimit/redis/deployment.yaml

```diff
@@ -0,0 +1,26 @@
+{{ if .Values.ratelimit.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: redis
+  namespace: {{ .Values.yorkie.namespace }}
```

Comment:
> How about moving ratelimit resources to `istio` namespace? Since ratelimit is applied to ingressgateway in `istio` namespace, it will be better to place them in `istio` namespace.

Same here.